src/firebase/firebase.js

```diff
@@ -0,0 +1,22 @@
+import * as firebase from 'firebase';
+
```

Comment:
Instead of using a wildcard import which would import everything from firebase, how about we just import only the part of firebase we need. Importing only the part of firebase we need can very well help us reduce the eventual bundle size of the applicaiton.

I came across this tutorial that explains how to import only the part of firebase you need, I believe it would be helpful: [link](https://medium.com/@zwacky/using-firebase-you-might-be-able-to-save-220kb-744269f845cc).
I also noticed that there is a section of firebase `npm` documentation that talks about this: [link](https://www.npmjs.com/package/firebase#include-only-the-features-you-need)

Reply:
This has been fixed as below:

```
import * as firebase from 'firebase/app';
import 'firebase/auth';
```